robottelo/config/__init__.py

```diff
@@ -1,4 +1,48 @@
-from robottelo.config.base import Settings
+import logging
 
-#: A :class:`Settings` object.
-settings = Settings()
+from dynaconf import LazySettings
+from dynaconf.validator import ValidationError
+
+from .validators import validators
+from robottelo.config.base import ImproperlyConfigured
+from robottelo.config.base import Settings as LegacySettings
+from robottelo.config.proxy import ConfigNodeWrapper
+from robottelo.config.proxy import ConfigProxy
+
+logger = logging.getLogger('robottelo.config')
+
+legacy_settings = LegacySettings()
+
+dynaconf_settings = LazySettings(
+    envvar_prefix="SATQE",
```

Comment:
SATQE? when this is ROBOTTELO

Reply:
I guess Bruno's intent was that Nailgun and Airgun will switch to Dynaconf as well one day, and they would all take config from single namespace.

I'll let others chime in before making change. Personally, I'm fine either way.

Reply:
I'd prefer ROBOTTELO as we would likely have repo-specific settings.

Reply:
I'm going to object to myself :-)

if we integrate dynaconf to our projects I can imagine situation where we are using different prefixes for differrent projects, but then we want one env var to represent the same entity in multiple projects we need to set multiple env vars....

Isn't better to just use one prefix everywhere, so now for all the projects better suites `SATQE` (than `ROBOTTELO`)

Reply:
To comply with https://github.com/SatelliteQE/robottelo/pull/7451 you have to use `ROBOTTELO_` prefix

This is already supported https://github.com/SatelliteQE/robottelo/blob/97c6845dd3bd86c8e034320736f7030125be6f0a/robottelo/config/base.py#L80
```
 value = os.environ.get(f'ROBOTTELO_{section.upper()}_{option.upper()}')
```

Reply:
I switched to ROBOTTELO, since that's people here seem to prefer.

@lpramuk #7451 is re-implementation of one of dynaconf functionalities, we only have it because we didn't had dynaconf. Once dynaconf is integrated, that PR should be reverted.
Also, does anybody use that feature? Jenkins sets these variables anywhere?

Reply:
Ah. OK. Jenkins didn't yet adopt it